src/main/external-player.js

```diff
@@ -17,12 +17,12 @@ let proc = null
 function checkInstall (playerPath, cb) {
   // check for VLC if external player has not been specified by the user
   // otherwise assume the player is installed
-  if (playerPath == null) return vlcCommand(cb)
+  if (playerPath.length === 0) return vlcCommand(cb)
```

Comment:
Can we simplify like this to cover both `null` and `empty string` just in case:
```suggestion
  if (!playerPath) return vlcCommand(cb)
```